app/src/main/java/com/codesoom/assignment/controller/CatToyController.java

```diff
@@ -0,0 +1,61 @@
+package com.codesoom.assignment.controller;
+
+import com.codesoom.assignment.application.CatToyService;
+import com.codesoom.assignment.domain.CatToy;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+
+/**
+ * 고양이 장난감에 대한 Http Request 요청 컨트롤러
+ */
+
+@RestController
```

Comment:
javadoc은 불특정 다수가 읽을 수 있다고 생각해 주세요. 항상 문장을 완성해 주는 것이 좋습니다. 실제로는 아무도 안 보고 나 혼자 보는 주석이라 해도 훈련한다고 생각하고 작성해 주세요. 설령 한솔님 본인도 앞으로 안 보게 된다 할지라도요.

```suggestion
/**
 * 고양이 장난감에 대한 Http Request 요청을 처리합니다.
 */
@RestController
```